
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=453166

# For some reason, rubyhead.swg is included without being needed (at least 
# for the current swig 1.3.33 version). This leads to a redefinition of 
# _mSWIG. The documentation of SWIG only talks about including just 
# the external_runtime file, not any additional includes. Also, the other
# bindings do not include anything from SWIG apart from the external 
# runtime. So I think this patch should suffice.  
#
#  -- Torsten Landschoff

Index: subversion/bindings/swig/ruby/libsvn_swig_ruby/swigutil_rb.c
===================================================================
--- subversion-1.4.4/subversion/bindings/swig/ruby/libsvn_swig_ruby/swigutil_rb.c.swig133
+++ subversion-1.4.4/subversion/bindings/swig/ruby/libsvn_swig_ruby/swigutil_rb.c
@@ -1,4 +1,3 @@
-#include "rubyhead.swg"
 #include "swig_ruby_external_runtime.swg"
 #include "swigutil_rb.h"
 #include <st.h>
